bundle/config/resources.go

```diff
@@ -168,3 +170,23 @@ func (r *Resources) Merge() error {
 	}
 	return nil
 }
+
+type ConfigResource interface {
+	Exists(ctx context.Context, w *databricks.WorkspaceClient, id string) bool
```

Comment:
This should return `(bool, error)`. The API call can fail and we should not make assertions on whether the resource exists or not in that case.